### PR TITLE
Update Hardware.rst

### PR DESCRIPTION
**Pull request by gyto6**
Repair website display and adding comment

### CODE REVIEW

File: docs/Performance and Tuning/Hardware.rst [hunk 1]
```diff
@@ -701,12 +701,17 @@ need to run TRIM on them. However, they cost more per GB than NAND flash
...
+   -  This one is more affordable refurbished and as powerfull that the P4800X in June 2022.
+   
+-  `Intel DC
    P1600X <https://www.servethehome.com/intel-optane-p1600x-small-capacity-ssd-for-boot-launched/>`__
+   -  This new model is the cheaper if bought unused but an excellent alternative in June 2022.
 
-   -  This gives basically the highest performance you can get as of
-      June 2020.
 
 Also, at time of writing in June 2020, only one model is listed. This is
```
Comment (gmelikov) [inline docs/Performance and Tuning/Hardware.rst:716]: What do you think about removing all additions per model, and this paragraph too (it became stale after your PR). So, looks like all these models may be great and people should choose based on their budget.

Comment (gyto6) [reply docs/Performance and Tuning/Hardware.rst:716]: Indeed, that's why'd prefer, but I tryed to stay humble in my contribution. 

I'll remove the comments and try to update the following paragraph to share your words.

Comment (gmelikov) [reply docs/Performance and Tuning/Hardware.rst:716]: I'm ok to just remove this paragraph, jfyi

**Comment by gyto6** [comment]
I removed all, and checked the preview, everything is fine this time concerning the shape.

**Comment by gyto6** [comment]
Ready to merge. Waiting for approval or remarks.

File: docs/Performance and Tuning/Hardware.rst [hunk 2]
```diff
@@ -701,17 +701,18 @@ need to run TRIM on them. However, they cost more per GB than NAND flash
...
    P1600X <https://www.servethehome.com/intel-optane-p1600x-small-capacity-ssd-for-boot-launched/>`__
 
-   -  This gives basically the highest performance you can get as of
-      June 2020.
-
-Also, at time of writing in June 2020, only one model is listed. This is
-due to there being few such drives on the market. The client models are
-likely to be outperformed by well configured NAND flash drives, so they
-have not been listed (although they are likely cheaper than NAND flash).
+Also, at time of writing in June 2020, only one model is listed.
```
Comment (gmelikov) [inline docs/Performance and Tuning/Hardware.rst:712]: This paragraph is still here, my thought was to drop it (we have more than one model now)

Comment (gyto6) [reply docs/Performance and Tuning/Hardware.rst]: Wasn't sure, I deleted the whole paragraph this time

**Review by gmelikov — APPROVED** [review]

**Pull request merged**